dart/lib/src/sentry_measurement_unit.dart

```diff
@@ -0,0 +1,53 @@
+enum SentryMeasurementUnit {
+  /// Nanosecond (`"nanosecond"`), 10^-9 seconds.
+  nanoSecond,
+
+  /// Microsecond (`"microsecond"`), 10^-6 seconds.
+  microSecond,
+
+  /// Millisecond (`"millisecond"`), 10^-3 seconds.
+  milliSecond,
+
+  /// Full second (`"second"`).
+  second,
+
+  /// Minute (`"minute"`), 60 seconds.
+  minute,
+
+  /// Hour (`"hour"`), 3600 seconds.
+  hour,
+
+  /// Day (`"day"`), 86,400 seconds.
+  day,
+
+  /// Week (`"week"`), 604,800 seconds.
+  week,
```

Comment:
`h`: Why do we have so many different units for time? Wouldn't it make sense to use `DateTime`?

Reply:
Its part of the protocol linked in the [meta issue](https://github.com/getsentry/team-mobile/issues/31) description.
https://github.com/getsentry/relay/blob/ed3a521f773ecd4bf9a2aefd7af80080d56d0841/relay-common/src/constants.rs#L353-L374